### DISCUSSION TITLE
how often do you run the script?

### DISCUSSION THREAD

**Discussion by rj-d2** [cross-seed/cross-seed#94] [Q&A]
Hello,

im interested in how often ppl run the script.

i have setup cross-seed to search all new torrents (added last 24 hours) every 30 minutes,
once per day it searches all torrents.

is this to often?
what do you think?

**Comment by Drazzilb08** [comment]
I run mines daily with a 5 day gap in between each torrent being looked at. This is with almost 10k torrents in my qBittorrent

**Reply by Soulplayer** [reply]
how do you run this daily?

**Reply by bakerboy448** [reply]
Crontab I assume

**Comment by afraazali** [comment]
I'm running mine every 10 minutes, however mine only looks at the last 3 days of torrents to cross seed, which at its highest is ever 15 torrents. I also limit the number of trackers it searches, which is a list of about 5.

**Reply by Soulplayer** [reply]
how do  you run this every 10 min?

**Reply by bakerboy448** [reply]
Crontab I assume

**Reply by Soulplayer** [reply]
I cannot get it to work. Do you have working example?

**Comment by mmgoodnow** [comment]
I run daemon mode with the webhook exclusively triggered by torrent completion.

**Comment by Soulplayer** [comment]
do you guys run this with crontab? If so which command do you guys use?
